ql/src/java/org/apache/hadoop/hive/ql/exec/tez/WorkloadManager.java

```diff
@@ -794,17 +828,17 @@ private void handleMoveSessionOnMasterThread(final MoveSession moveSession,
     final WmThreadSyncWork syncWork,
     final HashSet<String> poolsToRedistribute,
     final Map<WmTezSession, GetRequest> toReuse,
-    final Map<WmTezSession, WmEvent> recordMoveEvents) {
+    final Map<WmTezSession, WmEvent> recordMoveEvents, final boolean moveImmediately) {
```

Comment:
To keep it uniform with the new config, can we rename the variable as "delayedMove" instead of "moveImmediately" and use it accordingly?